pkg/minikube/cluster/cluster.go

```diff
@@ -315,3 +316,40 @@ func CreateSSHShell(api libmachine.API, args []string) error {
 	}
 	return client.Shell(strings.Join(args, " "))
 }
+
+func GetDashboardURL(api libmachine.API) (string, error) {
+	host, err := checkIfApiExistsAndLoad(api)
+	if err != nil {
+		return "", err
+	}
+
+	ip, err := host.Driver.GetIP()
+	if err != nil {
+		return "", err
+	}
+
+	port, err := getDashBoardPort()
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("http://%s:%s", ip, port), nil
+}
+
+func getDashBoardPort() (string, error) {
+	//$(kubectl get service kubernetes-dashboard --namespace=kube-system -o=jsonpath='{.spec.ports[0].nodePort}{"\n"}')
```

Comment:
Or this library looks easier to use: https://github.com/kubernetes/kubernetes/blob/dae5ac482861382e18b1e7b2943b1b7f333c6a2a/pkg/client/unversioned/clientcmd/doc.go

See the help text.